commands/run.go

```diff
@@ -522,6 +522,7 @@ func (h heightRange) divide(n int) []heightRange {
 	return subs
 }
 
+var mainnetGenesis = time.Date(2020, 8, 24, 22, 0, 0, 0, time.UTC)
 func estimateCurrentEpoch() int64 {
-	return int64(time.Since(time.Date(2020, 8, 24, 22, 0, 0, 0, time.UTC)) / (30 * time.Second))
+	return int64(time.Since(mainnetGenesis) / (builtin.EpochDurationSeconds))
```

Comment:
func estimateCurrentEpoch() int64 {
	return int64(time.Since(mainnetGenesis) / (builtin.EpochDurationSeconds * time.Second))
}

Reply:
Can you open an issue if you think there is a problem here